### PR TITLE
added tabbix easyonfigs

### PR DESCRIPTION
**Pull request by JensTimmerman**

### CODE REVIEW

**Comment by JensTimmerman** [comment]
@boegel good to go?

**Comment by boegel** [comment]
Jenkins: test this please

File: easybuild/easyconfigs/t/tabix/tabix-0.2.6-goolf-1.4.10.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock='MakeCp'
```
Comment (boegel) [inline easybuild/easyconfigs/t/tabix/tabix-0.2.6-goolf-1.4.10.eb]: needs style fix

File: easybuild/easyconfigs/t/tabix/tabix-0.2.6-intel-2014b.eb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+easyblock='MakeCp'
```
Comment (boegel) [inline easybuild/easyconfigs/t/tabix/tabix-0.2.6-intel-2014b.eb]: style fix pls

Comment (boegel) [inline easybuild/easyconfigs/t/tabix/tabix-0.2.6-intel-2014b.eb]: also, why not include header in both?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/f360e68c60abd6ecdc4a for a full test report.

**Comment by JensTimmerman** [comment]
@boegel good to go?

File: easybuild/easyconfigs/t/tabix/tabix-0.2.6-intel-2014b.eb [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+# Author: Jens Timmerman, Ghent University
+
+easyblock = 'MakeCp'
+
+name = 'tabix'
+version = '0.2.6'
+
+homepage = 'http://samtools.sourceforge.net'
+description = """ Generic indexer for TAB-delimited genome position files """
+toolchain = {'name': 'intel', 'version': '2014b'}
```
Comment (boegel) [inline easybuild/easyconfigs/t/tabix/tabix-0.2.6-intel-2014b.eb]: empty line above please

**Comment by JensTimmerman** [comment]
@boegel done

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Jenkins unresponsive? :(

**Comment by JensTimmerman** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
@JensTimmerman: I'm not sure why Jenkins isn't picking up the last commit, but we've seen this before...

Can you try adding another commit with something trivial?

**Comment by boegel** [comment]
Testing new stuff seems to work fine, see https://github.com/hpcugent/easybuild-easyconfigs/pull/1666

File: easybuild/easyconfigs/t/tabix/tabix-0.2.6-goolf-1.4.10.eb [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+files_to_copy = [
+    (["tabix", "bgzip", "tabix.py"], "bin"),
+    (["tabix.1"], "man/man1"),
+    "example.gtf.gz",
+    "example.gtf.gz.tbi",
+    "NEWS",
+    "ChangeLog"
+]
+buildopts = 'CFLAGS="-L$EBROOTZLIB/lib"'
```
Comment (boegel) [inline easybuild/easyconfigs/t/tabix/tabix-0.2.6-goolf-1.4.10.eb]: buildopts should go above files_to_copy

File: easybuild/easyconfigs/t/tabix/tabix-0.2.6-intel-2014b.eb [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+files_to_copy = [
+    (["tabix", "bgzip", "tabix.py"], "bin"),
+    (["tabix.1"], "man/man1"),
+    "example.gtf.gz",
+    "example.gtf.gz.tbi",
+    "NEWS",
+    "ChangeLog"
+]
+buildopts = 'CFLAGS="-L$EBROOTZLIB/lib"'
```
Comment (boegel) [inline easybuild/easyconfigs/t/tabix/tabix-0.2.6-intel-2014b.eb]: move up pls

**Comment by boegel** [comment]
Going in, thansk @JensTimmerman!

**Pull request merged**